helloworld.js

```diff
@@ -0,0 +1,2 @@
+
+console.log('hello, Ron Oh');
```

Comment:
Okay sorry about that

Ron 

 

From: Duncan Marsh [mailto:notifications@github.com] 
Sent: Tuesday, April 26, 2016 12:56 AM
To: codefellows-seattle-javascript-401n1/lab-00-install-node <lab-00-install-node@noreply.github.com>
Cc: Ronald Oh <ronoh2001@gmail.com>; Author <author@noreply.github.com>
Subject: Re: [codefellows-seattle-javascript-401n1/lab-00-install-node] update helloworld intro (#5)

 

In helloworld.js <https://github.com/codefellows-seattle-javascript-401n1/lab-00-install-node/pull/5#discussion_r61042390> :

> @@ -0,0 +1,2 @@
> +
> +console.log('hello, Ron Oh');

I See you have made two pull requests, If you decide to push a different branch up as your assignment, you should close the pull request you are not submitting
:)

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub <https://github.com/codefellows-seattle-javascript-401n1/lab-00-install-node/pull/5/files/90e88558aff2c6c4f98dd1886a88fa50489b631b#r61042390>   <https://github.com/notifications/beacon/AOpGlj2vpRmWqp6972o8G8v69eeleBquks5p7cUsgaJpZM4IPk-j.gif>